dev-libs/ferrisloki/files/ferrisloki-3.0.13-r3-configure-libsigc.patch

```diff
@@ -0,0 +1,95 @@
+Rollup of src_prepare() from previous revisions plus additional fixes
+
+ * derice damn configure script
+ * fix building with libsgc++-2.6
+ * fix building with /bin/sh -> dash (bug #886619)
+
+
+--- a/configure.in
++++ b/configure.in
+@@ -45,9 +45,6 @@ AC_STDC_HEADERS
+ #AC_ARG_PROGRAM
+ AM_PROG_LIBTOOL
+ 
+-STLPORT_CFLAGS=""
+-STLPORT_LDFLAGS=""
+-
+ ###############################################################################
+ ###############################################################################
+ ###############################################################################
+@@ -57,9 +54,9 @@ AM_FERRIS_SIGC2( 2.0.0 )
+ 
+ ###############################################################################
+ 
+-CFLAGS=" $STLPORT_CFLAGS $CXXFLAGS -DNDEBUG "
+-CXXFLAGS=" $STLPORT_CFLAGS $CXXFLAGS -DNDEBUG "
+-LDFLAGS=" $STLPORT_LDFLAGS $LDFLAGS"
++CFLAGS=" $CXXFLAGS -DNDEBUG "
++CXXFLAGS=" $CXXFLAGS -DNDEBUG "
++LDFLAGS=" $LDFLAGS"
+ AC_SUBST(CFLAGS)
+ AC_SUBST(CPPFLAGS)
+ AC_SUBST(LDFLAGS)
```

Comment:
and these ones too, they're always `AC_SUBST`'d implicitly

Reply:
done, good to know, thanks!